spacewalk/certs-tools/rhn_bootstrap_strings.py

```diff
@@ -407,6 +437,9 @@ def getRegistrationStackSh(saltEnabled):
 
     function getY_MISSING() {{
         local NEEDED="{PKG_NAME_YUM}"
+        if [ "0$VENV_ENABLED" -eq 1 ]; then
```

Comment:
You are comparing Integer with String. This may lead into warnings

```suggestion
        if [ $VENV_ENABLED  -eq 1 ]; then
```

Reply:
it's quite common solution for the case when the variable is not defined on time of checking.
I can fix it, but in this case need to add explicit setting of this variable to the top of the file otherwise we will get an error on evaluating `if [ -eq 1 ]; then`

Reply:
This is not correct as typical is, that you compare string with string.

```
   if ["x$VAR" = "x1"]; then
```

Is often used. I am also fine with this solution.

Reply:
yup, this is one of the options, but in this case I would prefer to use `if [ $VENV_ENABLED  -eq 1 ]; then` with explicit setting of `VENV_ENABLED` to `0` on top of the file, but it's better to check on the test system first.